usr.bin/proccontrol/proccontrol.1

```diff
@@ -77,6 +77,10 @@ Controls the KPTI enable, AMD64 only.
 .It Ar la48
 Control limiting usermode process address space to 48 bits of address,
 AMD64 only, on machines capable of 57-bit addressing.
+.It Ar cheri_revoke
+Control the use of CHERI revocation in the process.
+This is ultimately a hint to the runtime and some implementations may
```

Comment:
Having it as only a hint seems to defeat its purpose a bit? Unless you mean because it can be overridden?

Reply:
The kernel has no control over what userspace does about revocation. We can't make the runtime use it. (In principle we could drop MQ_*_FORCED and the kernel could enforce no revocation by making the system calls return ENOSYS, but that seems a bit silly). That being said, we could choose to document only the standard runtime and drop this bit.